azure-iot-device/azure/iot/device/iothub/transport/mqtt/mqtt_topic.py

```diff
@@ -64,24 +64,24 @@ def get_method_topic_for_subscribe():
     return "$iothub/methods/POST/#"
 
 
-def is_c2d_topic(topic):
+def is_c2d_topic(topic, device_id):
```

Comment:
I think this is a positive change, but is the function name still relevant? This now not just tells us if the topic is a c2d_topic, but also if it matches our device.

Reply:
I would like to revisit this whole module.  I'm going to add this as a topic when we talk next week.

---
In reply to: [287540129](https://github.com/Azure/azure-iot-sdk-python-preview/pull/94#discussion_r287540129) [](ancestors = 287540129)

Reply:
Er, I'd like to revisit iothub_mqtt_topic.py

---
In reply to: [287568462](https://github.com/Azure/azure-iot-sdk-python-preview/pull/94#discussion_r287568462) [](ancestors = 287568462,287540129)